pubsublite/consumer.go

```diff
@@ -228,6 +231,7 @@ func (c *Consumer) Run(ctx context.Context) error {
 				))
 				// Keep attempting to receive until a fatal error is received.
 				if errors.Is(err, pscompat.ErrBackendUnavailable) {
+					c.cfg.Logger.Info("transient error found, re-starting consumer...", zap.Error(err))
```

Comment:
We should have a span here. So we could also fetch it from context and add an event to it (so it's easier to search and match to a specific event)

Reply:
it seems strange to me to have a span here. `Receive()` may block for hours or days if no transient or fatal errors occur. Is this what you had in mind?

Reply:
Ah no, right. Somehow, I thought it returned the received object. Yeah, in that case, it makes sense. There shouldn't be any spans (we could create one to pass the log message, but that'd just be a different way to do logging).